code/modules/surgery/bodyparts/helpers.dm

```diff
@@ -48,6 +48,15 @@
 		which_hand = BODY_ZONE_PRECISE_R_HAND
 	return get_bodypart(check_zone(which_hand))
 
+/// Gets the inactive hand of the mob. Returns FALSE on non-carbons, otherwise returns the /obj/item/bodypart.
+/mob/proc/get_inactive_hand()
+	return FALSE
```

Comment:
false seems syntactically odd here compared to returning null, I guess it does not matter that much

Reply:
```suggestion
	return null
```